pkg/network/ebpf/c/prebuilt/http.c

```diff
@@ -53,9 +120,75 @@ int socket__http_filter(struct __sk_buff* skb) {
     return 0;
 }
 
+static __always_inline bool http2_marker_prefix(const char* buf) {
+    volatile bool match = buf[0]=='P' && buf[1]=='R' && buf[2]=='I' && buf[3]==' ' && buf[4]=='*' && buf[5]==' ' &&
+        buf[6]=='H' && buf[7]=='T' && buf[8]=='T';
+    return match;
+}
+
+static __always_inline bool http2_marker_suffix(const char* buf) {
+    volatile bool match = buf[0]=='P' && buf[1]=='/' && buf[2]=='2' && buf[3]=='.' &&
+        buf[4]=='0' && buf[5]=='\r' && buf[6]=='\n' && buf[7]=='\r' && buf[8]=='\n' && buf[9]=='S' &&
+        buf[10]=='M' && buf[11]=='\r' && buf[12]=='\n' && buf[13]=='\r' && buf[14]=='\n';
+    return match;
+}
```

Comment:
document and I think you can move it to `protocol-classification-helpers.h` and reuse those methods inside `is_http2`